src/main/asciidoc/configuring-cors.adoc

```diff
@@ -0,0 +1,71 @@
+[[customizing-sdr.configuring-cors]]
+= Configuring CORS
+
+For security reasons, browsers prohibit AJAX calls to resources residing outside the current origin. When working with client-side HTTP requests issued by a browser you want to enable specific HTTP resources to be accessible.
+
+Spring Data REST supports as of 2.6 http://en.wikipedia.org/wiki/Cross-origin_resource_sharing[Cross-origin resource sharing] (CORS) through http://docs.spring.io/spring/docs/{version}/spring-framework-reference/html/cors.html[Spring's CORS] support.
+
+
+== Repository interface CORS configuration
+
+You can add a `@CrossOrigin` annotation to your repository interfaces to enable CORS for the whole repository. By default `@CrossOrigin` allows origins and `HEAD` and `GET` HTTP methods:
```

Comment:
After discussing with @sdeleuze, `@CrossOrigin` on a repository will enable all HTTP methods by default. This default can be overridden. We can evaluate in a later step how we can provide the enabled repository methods to expose only the allowed CORS methods on a resource.